frontend/portals/SearchSuggestions/index.jsx

```diff
@@ -0,0 +1,124 @@
+import React, {
```

Comment:
![image](https://user-images.githubusercontent.com/5811286/82347410-d9e72f00-99f7-11ea-812f-92d984740a8b.png)

Reply:
It is a backend lie . I will put a check for actual product count

Reply:
did you add it?
![image](https://user-images.githubusercontent.com/5811286/82437323-790f3380-9a97-11ea-90c5-feacd87f1f54.png)

Reply:
loading spinners still spins forever

Reply:
I did: https://github.com/shopgate-professional-services/ext-search-suggestions-extended/pull/1/files#diff-655a2290f3c38806b891411d4e6fa0cdR28

What is response? I guess count is 566 but actual 1 item. I will add 1 more